.env.example

```
@@ -141,6 +141,20 @@ ENABLE_CUSTOM_PYTHON_FUNCTIONS=false
 TWILIO_ACCOUNT_SID=""
 TWILIO_AUTH_TOKEN=""
 
+# CRM outbound send (app/crm/connectivity). Per-merchant WhatsApp tokens and
+# phone number ids are NOT here — they are connector data, read from the vault
+# at send time. These only choose the endpoint and the deadline.
+#
+# Point the base URL at a local stub to exercise the whole dispatcher without
+# sending anything to Meta; seed a vault credential + installation + binding
+# by hand (migration 057 is the authoritative shape of the three rows).
+META_WHATSAPP_GRAPH_BASE_URL="https://graph.facebook.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)